.github/workflows/ci.yml

```
@@ -232,20 +193,96 @@
+        # Update Chart.yaml appVersion to match the Docker image we just built
+        sed -i "s/^appVersion: .*/appVersion: \"$VERSION\"/" charts/Chart.yaml
+        
+        # Update values.yaml image tag to reference the Docker image we just built
+        sed -i "s/tag: \".*\"/tag: \"$VERSION\"/" charts/values.yaml
+        
+        echo "Updated chart to reference Docker image: ghcr.io/vfarcic/dot-ai:$VERSION"
+    
+    - name: Package and push Helm chart
+      run: |
+        VERSION=${{ needs.version.outputs.new-version }}
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)